packages/vaadin-login/src/vaadin-login-mixin.d.ts

```diff
@@ -31,6 +31,11 @@ interface LoginMixin {
    */
   noForgotPassword: boolean;
 
+  /**
+   * If set, the user name field does not automatically receive focus when the component is attached to the document.
+   */
+  noAutoFocus: boolean;
```

Comment:
The property here is `noAutoFocus` but everywhere in JS it is called `noAutofocus`.
Please update it here for consistency.

Reply:
Done